src/QMCHamiltonians/ForceCeperley.cpp

```diff
@@ -216,9 +216,9 @@ std::unique_ptr<OperatorBase> ForceCeperley::makeClone(ParticleSet& qp, TrialWav
 //  void ForceCeperley::setObservables(PropertySetType& plist) {
 //    ///cerr << "ForceBase::setObs storing forces";
 //    int index = myIndex;
-//    for(int iat=0; iat<Nnuc; iat++) {
+//    for(int iat=0; iat<n_nuc_; iat++) {
 //      for(int x=0; x<OHMMS_DIM; x++) {
-//        plist[index] = forces(iat,x) + storeForces(iat,x);
+//        plist[index] = forces_(iat,x) + storeForces(iat,x);
 //        //cerr << " " << index << ":" << plist[index];
 //        index++;
 //      }
```

Comment:
Since these are commented out blocks it might be worth cleaning as well.

Reply:
Done